matminer/featurizers/composition.py

```diff
@@ -157,6 +158,54 @@ def implementors(self):
         return ["Jiming Chen", "Logan Ward", "Anubhav Jain"]
 
 
+class AtomicOrbitals(BaseFeaturizer):
+    '''
+    class to determine the highest occupied molecular orbital (HOMO) and
+    lowest unocupied molecular orbital LUMO in a composition. The atomic 
+    orbital energies of neutral ions with LDA DFT were computed by NIST.
+    https://www.nist.gov/pml/data/atomic-reference-data-electronic-structure-calculations
+    '''
+
+    def __init__(self):
```

Comment:
don't need this

Reply:
( referring to the empty __init__ method, not the reference to NIST - you can just remove the __init__ method altogether)